src/courses.json

```diff
@@ -26,15 +26,15 @@
   "Informationsmanagement": "IM",
   "Bachelor-Seminar: Digitale Hochschule: Aktuelle Trends und Herausforderungen": "Digitale Hochschule",
   "Betriebssysteme und Systemsoftware": "BS",
-  "Einführung in die Informatik ": "INFO",
+  "Einführung in die Informatik ": "EIDI",
```

Comment:
Actually, this is wrong.
![image](https://user-images.githubusercontent.com/26258709/165636888-c9860dce-377c-4e4b-a68f-22ec52985155.png)
`INFO II` != `EIDI II`
As far as I can remember This is `FPV`

Reply:
So what is what?

Do we then need
"Einführung in die Informatik 2" => "Info 2" and
"Einführung in die Informatik" => "EIDI"?

Reply:
I would prefer
```suggestion
  "Einführung in die Informatik 2": "FPV",
  "Einführung in die Informatik ": "EIDI",
```
Since this is what we called it.

Reply:
`FPV` is "Funktionale Programmierung und Verifikation". (IN0003)

Reply:
IN0003 is Einführung in die Informatik 2

https://campus.tum.de/tumonline/WBMODHB.wbShowMHBReadOnly?pKnotenNr=452808

Reply:
I'm talking about the image you sent above. This is another "Einführung in die Informatik 2".
It's this one: https://campus.tum.de/tumonline/ee/ui/ca2/app/desktop/#/slc.cm.reg/student/modules/detail/1528984/197
It's from the "Bachelor Geodesy and Geoinformation".

I have no idea what it contains as there is no information anywhere on this course but replacing it with FPV is wrong for sure.

Reply:
"Informally"?
![e4b3a49f8c27fe456e5dce19df30c4be](https://user-images.githubusercontent.com/64264938/169694616-4efba136-6355-44ea-b9e3-0d70475b4aab.png)
This is the official name

Reply:
https://campus.tum.de/tumonline/WBMODHB.wbShowMHBReadOnly?pKnotenNr=1458914 vs https://campus.tum.de/tumonline/WBMODHB.wbShowMHBReadOnly?pKnotenNr=452808

naming apparently depends upon your APSO..
<img src="https://media0.giphy.com/media/l3q2K5jinAlChoCLS/giphy.gif"/>

I dont know what is the best course of action.

Reply:
What about 
```suggestion
  "Einführung in die Informatik 2": "Einführung in die Informatik 2",
  "Einführung in die Informatik": "EIDI",
```
=> We don't touch the "2" but the "1" we rename.

Reply:
Looks like a good option.